hibernate-core/src/main/java/org/hibernate/internal/CoreMessageLogger.java

```diff
@@ -1792,4 +1792,8 @@ void attemptToAssociateProxyWithTwoOpenSessions(
 	@Message(value = "Association with `fetch=\"join\"`/`@Fetch(FetchMode.JOIN)` and `lazy=\"true\"`/`FetchType.LAZY` found. This will be interpreted as lazy: %s", id = 510)
 	void fetchModeJoinWithLazyWarning(String role);
 
+	@LogMessage(level = WARN)
+	@Message(value = "This version for [%s] is no longer supported, certain features may not work properly. Check the community dialects project for available legacy versions.", id = 511)
```

Comment:
Please also log the actual version and minimum version.

Reply:
:+1: